src/repository/noteRelations.repository.ts

```diff
@@ -67,4 +67,13 @@ export default class NoteRelationsRepository {
   public async hasRelation(noteId: NoteInternalId): Promise<boolean> {
     return await this.storage.hasRelation(noteId);
   }
+
+  /**
+   * Get all note parents based on note id
+   * @param noteId : note id to get all its parents
+   * @returns an array of note parents ids
+   */
+  public async getNoteParentsIds(noteId: NoteInternalId): Promise<NoteInternalId[]> {
+    return await this.storage.getAllNoteParentsIds(noteId);
```

Comment:
what is the meaning of "All" here? is it something different between regular `getNoteParentsIds` which is used in other places?